DfE.Hexagonal.Application/DfE.Hexagonal.Application.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Given the choice, my fairly strong preference is to avoid using implicit usings. 

What is the team's view on this?

Reply:
I gather this is on by default in .net 6 projects. This still has a configuration file that can be altered. It has the benefit of reducing the number of lines of Using at the start of a file, and the implied usings are only for those in the configuration, not every namespace used.
While there can be some issues with this, most of the time I think it will be ok, even if it feels wierd to not have to see or think about certain usings.